src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const amountPerDay = 40;
+  const discount3Days = 20;
+  const discount7Days = 50;
```

Comment:
Don't use numbers in var's naming, cause if you change the value in the future, you have to change var's name, as well